cooperator/models/subscription_request.py

```diff
@@ -123,10 +124,13 @@ def create(self, vals):
         subscription_request._send_confirmation_mail()
         return subscription_request
 
-    # todo: remove this and use create() instead. must stay in 12.0 (as
-    # deprecated) but be removed in 14.0.
     @api.model
     def create_comp_sub_req(self, vals):
+        warnings.warn(
+            "subscription.request.create_comp_sub_req() is deprecated. "
+            "please use .create() instead.",
+            DeprecationWarning,
+        )
```

Comment:
c'est un warning pour l'utilisateur ? Je pense qu'un warning dans les logs suffit.

Reply:
no, it should only appear as a log message, although i could not yet make it appear. it is used like this in some parts of odoo core as well as some oca modules. doc: https://docs.python.org/3/library/warnings.html#warnings.warn

Reply:
Ok then, TIL => thanks